denizen_scripts/survival/repo-link/spawn_scripts/tutorial.dsc

```diff
@@ -221,4 +223,4 @@ tutorial_events:
     on player exits spawn_cuboid flagged:tutorial:
       - injet tutorial_skipped
       - narrate "<&e>You have exited spawn, and stopped the tutorial."
-      - narrate "<&e>You can restart the tutorial at any time by using <&b>/tutorial"
\ No newline at end of file
+      - narrate "<&e>You can restart the tutorial at any time by using <&b>/tutorial"
```

Comment:
Could this line benefit from punctuation?

Reply:
TBH I didnt even see the stuff at the bottom yet, i was just working on holograms first.